access_point/server/config.sh

```diff
@@ -1,8 +1,12 @@
 #! /bin/bash
+
+SCRIPT_PATH=$(realpath $0)
+DIR_PATH=$(dirname $SCRIPT_PATH)
+
 STT="$1"
 TTS="$2"
 HOTWORD="$3"
 WAKE="$4"
 
-cd /home/pi/SUSI.AI/susi_linux
+cd $DIR_PATH/../../
```

Comment:
Should find a better way than "cd" around!

Reply:
maybe true but in the context of atomic changes step by step it is valid to leave that thought out of this commit.